README.md

```diff
@@ -44,72 +44,68 @@ After you're finished please remove all the comments and instructions!
     - [Key Features](#key-features)
   - [🚀 Live Demo](#live-demo)
 - [💻 Getting Started](#getting-started)
-  - [Setup](#setup)
-  - [Prerequisites](#prerequisites)
-  - [Install](#install)
-  - [Usage](#usage)
-  - [Run tests](#run-tests)
-  - [Deployment](#triangular_flag_on_post-deployment)
-- [👥 Authors](#authors)
-- [🔭 Future Features](#future-features)
-- [🤝 Contributing](#contributing)
-- [⭐️ Show your support](#support)
-- [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
-- [📝 License](#license)
```

Comment:
- [ ] Your readme ended at the "Getting Started" section. Kindly add the remaining links as we have on the template and add their corresponding sections.

  Note that only the FAQ section is optional, the rest are mandatory.

  ![iot](https://user-images.githubusercontent.com/81468675/217319446-467ca5d4-0228-4d46-a3ce-a6db982927ad.jpg)